src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/InternalMetadataProviderIdentityExtensions.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Is this identical to the Spdx22SbomParser copy of the file? Can we move it to a common location and pull from there instead of duplicating?

Reply:
Same question for SPDXExtensions.cs.

Reply:
I moved all but one of these methods (which is not shared by the generators) into shared file. For spdxExtensions all the methods are different so I kept them as is